contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -5550,7 +5550,12 @@ makeFetchCursorStatement(TSqlParser::Fetch_cursorContext *ctx)
 	/* cursor_name */
 	auto targetText = ::getFullText(ctx->cursor_name());
 	result->curvar = lookup_cursor_variable(targetText.c_str())->dno;
+    
+	/* FETCH CURSOR without destination should be blocked inside a function. */
 
+	if (is_compiling_create_function() && !ctx->INTO()){
```

Comment:
this has potential for regression if SQL server function allows certain selects without INTO clause. How are we making sure that that is not the case?

Reply:
This is Fetch cursor syntax and returns set of rows. If INTO clause is not specified it returns results to client. Adding few more tests.

Reply:
We can probably include test cases like `SELECT @var = col` & `SELECT ... INTO ...` inside functions